.github/workflows/deploy-to-test-and-live.yml

```diff
@@ -0,0 +1,78 @@
+# Github action to deploy to Pantheon
+name: Deploy to Pantheon Test and Live
+
+on:
+  release:
+    types: [published]  # Trigger when a release is published
+
+jobs:
+  deploy-to-test:
+    environment: Test
+    permissions:
+      contents: read
+      deployments: write
+      pull-requests: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.release.tag_name }}  # Checkout the release tag (e.g., v1.0.0)
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.4"
+
+      - name: Install Terminus
+        uses: pantheon-systems/terminus-github-actions@v1
+        with:
+          pantheon-machine-token: ${{ secrets.PANTHEON_MACHINE_TOKEN }}
+          disable-cache: true
+
+      - name: Prepare deployment message
+        run: |
+          RELEASE_BODY=$(echo "${{ github.event.release.body }}")
+          echo "DEPLOY_TEST_MESSAGE<<EOF" >> $GITHUB_ENV
+          echo "$RELEASE_BODY" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
What is $GITHUB_ENV used for?

Reply:
It appears to be [a temp file](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable) when the job runs. This allows environment variables to be shared across steps.